app/controllers/application_controller.rb

```diff
@@ -80,7 +80,7 @@ def require_user
       unless current_user
         store_location
         flash[:notice] = "You must be logged in to access this page"
-        redirect_to login_url+'?return_to=' + URI.encode(request.env['PATH_INFO'])
```

Comment:
Actually the `store_location` method already stores the full path of the request in the session. See reference here: https://github.com/publiclab/plots2/pull/550/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eL98. The `URI.encode(request.env['PATH_INFO'])` was actually returning the parameters but it was only available to `\login` page. As soon it posted the login parameters of an user the parameters were no longer available and it didn't redirect to with no parameters in the `/post` url. So it was an issue that I solved it here using the changes done in the user_sessions_controller: https://github.com/publiclab/plots2/pull/550/files#diff-0770692f348632e109afe78e3585bfe0R35